### ISSUE TITLE
Doesn't work in some AWS account environments on v1.3.2

### ISSUE THREAD

**Issue by yaggytter** [yaggytter/chrome-extension-awssso#23]
got following error messages:
```
content.js:159 Uncaught TypeError: Cannot read properties of null (reading 'querySelectorAll')
    at content.js:159:9
    at waitForElement (content.js:42:9)
```

I guess the timing of DOM creation was changed.
I will fix it ASAP.

**Comment by yaggytter** [comment]
regarding
https://github.com/yaggytter/chrome-extension-awssso/issues/22
https://github.com/yaggytter/chrome-extension-awssso/issues/21

**Comment by yaggytter** [comment]
Fixed it and will release it as v1.3.3

https://github.com/yaggytter/chrome-extension-awssso/pull/24

**Comment by yaggytter** [comment]
released it.
